lib/src/util/logging.cc

```diff
@@ -101,7 +101,6 @@ static void setupLoggingImp(std::ostream& log_stream,
         sink->set_filter(expr::has_attr(access_outcome));
         auto core = boost::log::core::get();
         core->add_sink(sink);
-
```

Comment:
Also, just noticed line 100 is terribly dangerous. You can't just move a std::shared_ptr to a boost::shared_ptr, memory ownership gets all screwed up.

Nevermind, missed the implicit constructor call. However, related to LTH-115 you'll want to explicitly call the `access_writer` constructor.